### PR TITLE
strategies.access_token missing in configuration JSON schema

### PR DESCRIPTION
**Pull request by trecloux**
## Proposed changes
Configuration JSON schema did not include `strategies.access_token`.

## Checklist
- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: .schema/config.schema.json [hunk 1]
```diff
@@ -612,6 +612,14 @@
...
           ],
           "default": "wildcard"
+        },
+        "access_token": {
+          "type": "string",
+          "description": "Defines access token type. jwt is a bad idea, see https://www.ory.sh/docs/hydra/advanced#json-web-tokens",
+          "enum": [
+            "opaque",
+            "jwt"
+          ]
```
Comment (aeneasr) [inline .schema/config.schema.json:622]:
```suggestion
            "default": "opaque",
          ]
```

**Comment by aeneasr** [comment]
Oh and could you please add a test for `https://github.com/ory/hydra/blob/master/driver/configuration/provider.go#L33` here: https://github.com/ory/hydra/blob/master/driver/configuration/provider_viper_test.go#L214 ? :)

**Comment by trecloux** [comment]
I added the test, thanks for the hint.

Do you want me to squash my commits ?

**Comment by aeneasr** [comment]
No that's fine! Thank you :)

**Pull request merged**